changelog/@unreleased/pr-262.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: feature
+feature:
+  description: |-
+    TCP writer uses a buffered channel to queue messages
```

Comment:
If this was motivated by a particular issue (performance/configuration or otherwise), can you add a blurb on that here so it's clear why this would matter when reviewing the release notes?